docs/hive-integration/import-installed-cluster.md

```diff
@@ -1,84 +1,169 @@
-# Import Installed Cluster
+# Import an existing OCP cluster as an Assisted Day 2 Cluster
 
-It is possible to import an existing installed OpenShift in order to be able to add more workers to it. (Day 2 operation)
+It is possible to take an existing OCP cluster and import this as a Day2 cluster into assisted so that additional workers may be added.
```

Comment:
We're also starting to allow adding control plane nodes, so better to make it generic.  

```suggestion
It is possible to import an existing OCP cluster so that additional nodes may be added.
```